src/liquid/components/ld-sidenav/ld-sidenav-back/ld-sidenav-back.css

```diff
@@ -0,0 +1,135 @@
+.ld-sidenav-back {
+  /* layout */
+  --ld-sidenav-back-border-radius: var(--ld-br-l);
+  --ld-sidenav-back-icon-size: var(--ld-sp-40);
+
+  /* colors */
+  --ld-sidenav-back-col: var(--ld-col-neutral-800);
+  --ld-sidenav-back-indicator-col: transparent;
+
+  outline: none;
+  display: block;
+
+  @media (hover: hover) {
+    &:where(:hover) {
+      --ld-sidenav-back-col: var(--ld-thm-primary);
+      --ld-sidenav-back-indicator-col: var(--ld-col-neutral-300);
+    }
+  }
+
+  &:focus:focus-visible {
+    --ld-sidenav-back-col: var(--ld-thm-primary);
+    --ld-sidenav-back-indicator-col: var(--ld-thm-primary-focus);
+  }
+
+  &:active {
+    --ld-sidenav-back-col: var(--ld-thm-primary-active);
+  }
+
+  &--rounded {
+    --ld-sidenav-back-border-radius: calc(
+      0.5 * var(--ld-sidenav-back-icon-size)
+    );
+  }
+
+  ::slotted(ld-sidenav-navitem) {
+    margin-top: var(--ld-sidenav-padding-y);
+    margin-bottom: var(--ld-sidenav-padding-y);
+  }
+}
+
+.ld-sidenav-back__btn-back {
+  display: none;
+  width: calc(100% - 2 * var(--ld-sidenav-padding-x));
+  background-color: transparent;
+  position: relative;
+  font: var(--ld-typo-body-s);
+  border: 0;
+  border-radius: var(--ld-sidenav-back-border-radius);
+  cursor: pointer;
+  user-select: none;
+  touch-action: manipulation;
+  color: var(--ld-sidenav-back, var(--ld-col-neutral-800));
+  grid-template-columns: var(--ld-sidenav-back-icon-size) 1fr;
+  gap: var(--ld-sp-12);
+  align-items: center;
+  font-weight: 700;
+  line-height: 1;
+  box-sizing: border-box;
+  padding: 0;
+  text-align: left;
+  outline: none;
+  margin: var(--ld-sidenav-padding-y) var(--ld-sidenav-padding-x);
+  -webkit-touch-callout: none;
+
+  &::before {
+    content: '';
+    position: absolute;
+    inset: calc(-1 * var(--ld-sp-6));
+    border-radius: calc(var(--ld-sidenav-back-border-radius) + var(--ld-sp-6));
```

Comment:
This is how it has been designed by @NicolasBlock91. The reason here is that there is an element with a border radius and another wrapping element around that element, with a border-radius that needs to have a size to have "parallel" borders with the contained element. The calculation takes into account the inset. I wouldn't put it into a variable as this would cover up this fact.